tests/test_integration.py

```diff
@@ -1103,7 +1112,16 @@ def test_pex_source_bundling():
                 )
 
             pex_path = os.path.join(output_dir, "pex1.pex")
-            res = run_pex_command(["-o", pex_path, "-D", input_dir, "-e", "exe",])
+            res = run_pex_command(
+                [
+                    "-o",
+                    pex_path,
+                    "-D",
+                    input_dir,
+                    "-e",
+                    "exe",
+                ]
+            )
```

Comment:
FYI this is only because of the trailing comma. If you removed it, Black would keep things on the same line.

Not a big deal either way - only wanted to explain the new algorithm.

Reply:
OK, thanks. I'll let this stand.